Loading checkpoint from "/data/xhj/model/v2/best.model"

99
0.00032712842318033104
1450339
********Starting metrics evaluation on test dataset**********
/home/xhj/src/eval_composite.py:360: RuntimeWarning: invalid value encountered in log
  log_ = np.log(numerator / denominator)
Numerator: [[0.00284424]]
Denominator: [[-2.3382046e-05]]
Numerator: [[0.0001735]]
Denominator: [[-3.4215645e-05]]
Numerator: [[0.00362766]]
Denominator: [[-0.00010177]]
Numerator: [[0.00110443]]
Denominator: [[-0.00017934]]
Numerator: [[0.00133597]]
Denominator: [[-0.00055303]]
Numerator: [[0.00394703]]
Denominator: [[-0.00165695]]
Numerator: [[0.00636382]]
Denominator: [[-4.259846e-05]]
Numerator: [[0.0002333]]
Denominator: [[-9.481683e-05]]
Numerator: [[0.00140741]]
Denominator: [[-0.0021542]]
Numerator: [[0.00203206]]
Denominator: [[-0.00036474]]
Numerator: [[0.0018856]]
Denominator: [[-0.00194905]]
Numerator: [[0.00410831]]
Denominator: [[-0.00060675]]
Numerator: [[0.00262296]]
Denominator: [[-0.00011474]]
Numerator: [[0.0041685]]
Denominator: [[-0.00223267]]
Numerator: [[0.00554421]]
Denominator: [[-0.00010729]]
Numerator: [[0.01188304]]
Denominator: [[-0.00211089]]
STOI: 0.9359
SSNR: 5.4659
PESQ: 2.9577
CSIG: 4.3294
CBAK: 3.2655
COVL: 3.6615
